.github/workflows/build-test-package.yml

```diff
@@ -3,9 +3,9 @@ name: Build, test, package
 on: [push,pull_request]
 
 env:
-  itk-git-tag: "v5.4rc04"
-  itk-wheel-tag: "v5.4rc04"
-  itk-python-package-tag: "v5.4rc04"
+  itk-git-tag: "v5.4.0"
+  itk-wheel-tag: "v5.4.0"
+  itk-python-package-tag: "v5.4.0"
```

Comment:
I would suggest to use InsightSoftwareConsortium/ITKPythonPackage@f38e8b0f38580807809b0412235fb2ba0946deba instead to benefit from post v5.4.0 fixes. They have been useful for preparing RTK release!